File: check_businesses.py

import requests
try:
    response = requests.get('http://localhost:3000/api/businesses')
    businesses = response.json()
    for b in businesses:
        print(f"Name: {b.get('name')}, Image: {b.get('image')}")
except Exception as e:
    print(e)

File: check_businesses.js

const http = require('http');

http.get('http://localhost:3000/api/businesses', (resp) => {
  let data = '';

  resp.on('data', (chunk) => {
    data += chunk;
  });

  resp.on('end', () => {
    try {
        const businesses = JSON.parse(data);
        businesses.forEach(b => {
            console.log(`Name: ${b.name}, Image: ${b.image}`);
        });
    } catch (e) {
        console.error(e.message);
    }
  });

}).on("error", (err) => {
  console.log("Error: " + err.message);
});
